components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/java/org/wso2/carbon/user/mgt/ui/UserAdminClient.java

```diff
@@ -258,7 +276,22 @@ public void addRemoveUsersOfRole(String roleName, String[] newUsers, String[] de
         } catch (Exception e) {
             handleException(e);
         }
-        
+
+        return new FlaggedName[0];
+    }
+
+    public FlaggedName[] listUserByClaimWithPermission(ClaimValue claimValue, String filter, String permission, int
+            limit) throws AxisFault {
+        try {
+            org.wso2.carbon.user.mgt.stub.types.carbon.ClaimValue newClaimValue = new
```

Comment:
FQN of the class is used here. Are there other classes with the same name used in this code?